.lagoon/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN npm install -g pnpm@8.6.12 && pnpm config set store-dir /tmp/cache/pnpm
 
 # Copy pnpm lockfile and install dependencies.
 COPY pnpm-lock.yaml .npmrc /app/
-#COPY patches /app/patches
+# COPY patches /app/patches
```

Comment:
Why don't we commit a `.gitkeep` file in `/patches` directory and remove this comment so we never need to update this Dockerfile for projects which eventually get local (custom) patches?